web/docs/troubleshooting.md

```diff
@@ -0,0 +1,39 @@
+# Troubleshooting
+
+Sometimes things go wrong. Oh no :-(. This page provides guidance on what to do
+in error scenarios we've encountered.
+
+:::tip Get help!
+Need someone to talk to? Swing by our [Discord](/discord) channel where the
+Tenzir team and the community hang out to help each other. Alternatively, send
+us an email at support@tenzir.com. We'll help you out as soon as possible.
+:::
+
+## Connectivity
+
+### A node does not connect to the platform
+
+After you've followed the instructions to [deploy a
+node](setup-guides/deploy-a-node.md), the node does not show up in the platform.
+
+Here's what you can do:
+
+1. Ensure that your firewall allows outbound 443/TCP traffic.
+2. Start the node manually on the command line via `tenzir-node` and observe the
+   output. In case you see a warning or an error, share it with us.
+
+### A node fails to connect with `system_error: failed to resolve`
+
+You may see this error message when a node attempts to connect to the platform:
+
+```
+platform-client failed to connect: !! system_error: failed to connect: !! system_error: failed to resolve; will retry in 2m
```

Comment:
Would you mind changing this error message to remove the nested `caf::error` instances, and replace them with a proper diagnostic via `diagnostic::error(...).to_error()`, that you can then use in this warning message and also reference here? 

If you don't want to change this now, then please create a follow-up issue that also references this troubleshooting section, because the error message is not helping the user at all right now.

Reply:
Done. Can you please check https://github.com/tenzir/tenzir-plugins/pull/227 as well? I bumped the submodule pointer in this PR.